### ISSUE TITLE
loss does not converge

### ISSUE THREAD

**Issue by Inception41** [nftblackmagic/Diffusion-Tryon-Trainer#9]
hello，this is a very good job, but when I train it, the loss function dose not converge.  I use the command just like the train_ootd.sh,but comment the 8bit_adam and mixed_precision='fp16' ,and the batch size is 1. Are your training settings the same as in train_ootd.sh?
